src/tt.cpp

```diff
@@ -48,6 +47,8 @@ void TranspositionTable::resize(size_t mbSize) {
                 << "MB for transposition table." << std::endl;
       exit(EXIT_FAILURE);
   }
+  memset(mem, 0, clusterCount * sizeof(Cluster) + CacheLineSize - 1);
```

Comment:
I'm not sure about that
at least until now calloc would clean whole mem 
clear cleans table not mem. seems like table is kind of shifted one